examples/make-ps-file.py

```diff
@@ -0,0 +1,60 @@
+import sys
+import numpy as np
+import segyio
+from itertools import izip as izip
+
+def main():
+    if len(sys.argv) < 9:
+        sys.exit(" ".join( "Usage: {} [file] [samples]",
+                           "[first iline] [last iline]",
+                           "[first xline] [last xline]",
+                           "([first offset] [last offset])"
+                         ).format(sys.argv[0]))
+
+    spec = segyio.spec()
+    filename = sys.argv[1]
+
+# to create a file from nothing, we need to tell segyio about the structure of
+# the file, i.e. its inline numbers, crossline numbers, etc. You can also add
+# more structural information, This is the absolute minimal specification for a
+# N-by-M volume with K offsets volume
+    spec.sorting = 2
+    spec.format = 1
+    spec.samples = int(sys.argv[2])
+    spec.ilines = range(*map(int, sys.argv[3:5]))
+    spec.xlines = range(*map(int, sys.argv[5:7]))
+    spec.offsets = range(*map(int, sys.argv[7:9]))
+    if len(spec.offsets) == 0: spec.offsets = [1]
+
+    with segyio.create(filename, spec) as f:
+        # one inline consists of 50 traces
+        # which in turn consists of 2000 samples
+        start = 0.0
+        step = 0.00001
+        # fill a trace with predictable values: left-of-comma is the inline
+        # number. Immediately right of comma is the crossline number
+        # the rightmost digits is the index of the sample in that trace meaning
+        # looking up an inline's i's jth crosslines' k should be roughly equal
+        # to i.j0k. Offset values are multiplied by 100
+        trace = np.arange(start = start,
+                          stop  = start + step * spec.samples,
+                          step  = step,
+                          dtype = np.single)
+
+        # one inline is N traces concatenated. We fill in the xline number
+        line = np.concatenate([trace + (xl / 100.0) for xl in spec.xlines])
+        line = line.reshape( (len(spec.xlines), spec.samples) )
+
+        xlines, offsets  = len(spec.xlines), len(spec.offsets)
```

Comment:
xlines vs spec.xlines have different meaning. should be more explicit. e.g. xlines_count or something.